test/controllers/images_controller_test.rb

```
@@ -37,4 +37,20 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
       get image_path(id: -1)
     end
   end
+
+  test 'root link to image index' do
+    get root_path
+    assert_response :ok
+    assert_select "a[href='/images/new']", count: 1, text: 'Add Image'
+  end
+
+  test 'should get index' do
+    Image.create(url: 'https://www.google.com/test1.jpg')
+    Image.create(url: 'https://www.google.com/test2.jpg')
```

Oh, I missed some things. 

It can be good to do `Image.create!` instead of `Image.create` as the former will throw an exception if the `create` fails and the later will just fail silently.